src/Pages/Task_index.js

```diff
@@ -1,11 +1,51 @@
 import React, { Component } from 'react';
+import { getTasks } from '../API'
+import AuthService from '../services'
+import Task_Card from '../Components/Task_card'
+import { Table } from 'react-bootstrap';
+
+
 
 class Task_Index extends Component {
+  constructor(props) {
+    super(props)
+    this.auth = new AuthService()
```

Comment:
auth isn't being used anywhere in this component, so do we need to instantiate it here?